traffic_ops/app/lib/UI/Topology.pm

```diff
@@ -554,52 +554,30 @@ sub gen_crconfig_json {
     return ($data_obj);
 }
 
-sub read_crconfig_json {
```

Comment:
line 578 is basically all that is needed to "read" now - https://github.com/apache/incubator-trafficcontrol/pull/131/files#diff-42a47b5d7e269e8d875ab706652b0f70R578